apps/studio/src/lib/editor/engine/canvas/index.ts

```diff
@@ -61,8 +61,16 @@ export class CanvasManager {
         this.saveSettings();
     }
 
-    get frames() {
-        return this.webFrames;
+    get frames(): (FrameSettings & { folderPath: string })[] {
```

Comment:
I would not have done it like this. There are better ways to get the folder path.